examples/reference-entity-model/entities/BabyRegistry.entity.json

```diff
@@ -0,0 +1,34 @@
+{
+  "info": {
+    "title": "BabyRegistry",
+    "version": "0.0.1",
+    "baseUri": "http://marklogic.com/example/"
+  },
+  "definitions": {
+    "BabyRegistry": {
+      "primaryKey": "babyRegistryId",
+      "properties": {
+        "babyRegistryId": {
+          "datatype": "integer"
+        },
+        "arrivalDate": {
+          "datatype": "date"
+        },
+        "ownedBy": {
+          "datatype": "integer",
+          "relatedEntityType": "http://example.org/Customer-0.0.1/Customer",
+          "joinPropertyName": "customerId"
+        },
+        "babyRegistryIncludes": {
```

Comment:
Let's call this "hasProduct" so that the property identifies the type of property; "babyRegistryIncludes" is redundant because we already know this is being included by a baby registry by virtue of being a BabyRegistry property.